spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/RabbitListenerContainerFactory.java

```diff
@@ -48,4 +49,17 @@ default C createListenerContainer() {
 		return createListenerContainer(null);
 	}
 
+	/**
+	 * set a bean name.
+	 * @since 3.2
+	 */
```

Comment:
We don't need this JavaDoc since the method is inherited from the `BeanNameAware`.
However it still have to come with an `@Override`